### ISSUE TITLE
Support for GL_NEAREST drawing of sprites

### ISSUE THREAD

**Issue by TerrySoba** [pvcraven/arcade#405]
## Enhancement request:
It would be nice if it was possible to draw sprites without smoothing when scaled. This could be achieved by using GL_NEAREST. This would be really nice for retro games with pixelated graphics.
I had a look at the file **sprite_list.py** and it already had the code to enable nearest neighbor filtering, but it was commented out:
```
# gl.glTexParameterf(gl.GL_TEXTURE_2D, gl.GL_TEXTURE_MIN_FILTER, gl.GL_NEAREST)
# gl.glTexParameterf(gl.GL_TEXTURE_2D, gl.GL_TEXTURE_MAG_FILTER, gl.GL_NEAREST)
```

### What should be added/changed?

Maybe the **draw()** method of the class **SpriteList** could be extended with a filter parameter. Something like this:
```
def draw(self, **kwargs):
    [...]
    if "filter" in kwargs:
        gl.glTexParameterf(gl.GL_TEXTURE_2D, gl.GL_TEXTURE_MIN_FILTER, kwargs["filter"])
        gl.glTexParameterf(gl.GL_TEXTURE_2D, gl.GL_TEXTURE_MAG_FILTER, kwargs["filter"])
```
That way the filtering would be configurable.

**Comment by pvcraven** [comment]
Change in code, ready for next release.

**Comment by pvcraven** [comment]
Fixed with release 2.2.6.
